nodes/src/nodes/tool_github/IInstance.py

```
@@ -0,0 +1,1034 @@
+    if 'input' in input_obj and isinstance(input_obj.get('input'), dict):
+        inner = input_obj['input']
+        extras = {k: v for k, v in input_obj.items() if k != 'input'}
+        input_obj = {**inner, **extras}
+    input_obj.pop('security_context', None)
+    return input_obj
+
+
+def _require_str(args: dict, key: str, fn: str) -> str:
+    val = (args.get(key) or '').strip()
+    if not val:
+        raise ValueError(f'{fn}: "{key}" is required')
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)